assets/scss/_fonts.scss

```diff
@@ -1,50 +1,44 @@
 @font-face {
-  font-family: 'Inter UI';
```

Comment:
Why did you change the font?

Reply:
See https://github.com/rhazdon/hugo-theme-hello-friend-ng/pull/62#discussion_r327017467